salt_resources/states/top.sls

```diff
@@ -17,13 +17,15 @@ base:
     # - nginx                   {# requires {{ grains['dvcs'] }}.repo #}
     # - venv                    {# requires {{ grains['dvcs'] }}.repo, python #}
     # - venv.pip_requirements   {# requires venv #}
-    # - conda                   {# requires users #}
-    # - conda.anaconda          {# requires conda #}
+    - conda                   {# requires users #}
```

Comment:
These should all be commented out in rambo master. I'll make that change after testing this.